src/components/App/App.js

```diff
@@ -30,6 +31,7 @@ class App extends Component {
     return (
       <Router>
         <Nav />
+        <MobileNav />
```

Comment:
Does this only render on mobile?

Reply:
Yes!

Reply:
I could put a comment there for clarity. At the bottom of App.css I set it to hide whichever one isn’t in use